cmd/edgectl/aes_install_errors.go

```diff
@@ -42,39 +45,51 @@ func (i *Installer) NoKubectlError(err error) Result {
 // Unable to get cluster information
 func (i *Installer) NoClusterError(err error) Result {
 	url := "https://www.getambassador.io/docs/latest/topics/install/help/no-cluster"
+
 	return Result{
 		Report:       "fail_no_cluster",
 		URL:          url,
-		ShortMessage: "Could not find a Kubernetes cluster",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about how to setup up your Kubernetes environment to continue installing Ambassador Edge Stack.", url),
+		ShortMessage: "The installer could not find a Kubernetes cluster",
+		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about how to set up your Kubernetes environment to continue installing Ambassador Edge Stack.", url),
 		Err:          err,
 	}
 }
 
 // Unable to get client configuration or namespace
 func (i *Installer) GetRestConfigError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/get-rest-config"
+
 	return Result{
-		Report: "fail_no_cluster",
-		URL:    "https://www.getambassador.io/docs/latest/topics/install/help/get-rest-config",
-		Err:    err,
+		Report:       "fail_no_cluster",
+		ShortMessage: "The installer could not communicate with your Kubernetes cluster",
```

Comment:
This short message is generic (like the page it references) but should ideally be a little more specific about the particular error case, IMO. In this case, the installer was unable to get the client configuration or namespace from the Kubernetes cluster...

Reply:
Same/similar comment applies to other messages below where you've included a generic short message.